theme/ckeditor5-ui/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+@import "../../../mixins/_rounded.css";
+
+:root {
+	--ck-tooltip-arrow-size: 5px;
+}
+
+.ck-tooltip {
+	left: 50%;
+
+	/**
+	 * A class once applied displays the tooltip south of the element.
+	 *
+	 *       [element]
+	 *           ^
+	 *     +-----------+
+	 *     |  Tooltip  |
+	 *     +-----------+
+	 */
+	&.ck-tooltip_s {
+		bottom: calc(-1 * var(--ck-tooltip-arrow-size));
+		transform: translateY( 100% );
+
+		& .ck-tooltip__text::after {
+			top: calc(-1 * var(--ck-tooltip-arrow-size));
+			transform: translateX( -50% );
```

Comment:
Let's speed up that transform: with `translate3d( -50%, 0, 0);`

Reply:
👉 https://github.com/ckeditor/ckeditor5-theme-lark/pull/114#discussion_r152842953